### ISSUE TITLE
Fields sometimes null after retrieving QueryResult from InfluxDB and conversion to POJO

### ISSUE THREAD

**Issue by bennycwlau** [influxdata/influxdb-java#733]
Most of the time I'm able to successfully query my InfluxDB and convert the result to POJO without any issue. However, sometimes after converting the query result to a POJO, some of the property fields in the POJO are null even though none of the fields are null/empty for that record in my InfluxDB measurement. This only happens to a few of the property fields too, not all of them. Why does this happen? 

My Spring Boot application also receives many concurrent requests at any given moment, which makes me think this may be a multi-threading issue. Also my Transaction class has a @Data Lombok annotation if that is relevant. I have an example below for my Transaction measurement/pojo.

**Transaction measurement (in InfluxDB)**
Time: 1612283422708733547
Amount: 18.20
Status: Success

**POJO (in Spring Boot application after resultMapper.toPOJO)**
Time: 1612283422708733547
Amount: null
Status: null

Current logic in application

```
Query query = new Query(String.format(queryString, lastTimestamp), "transaction");
QueryResult queryResult = influxDB.query(query);
return resultMapper.toPOJO(queryResult, Transaction.class, TimeUnit.NANOSECONDS);
```

InfluxDB version: 1.8.3

Could this be related to #676 or #693 or #683 ?

**Comment by bennycwlau** [comment]
@majst01 Any ideas?

**Comment by majst01** [comment]
No, is suspect a concurrency issue in some way. We had never this type of error in the past.
You can enable debug logging and see where the data is lost.
